doc/api/errors.md

```diff
@@ -647,144 +644,129 @@ urlSearchParams.has.call(buf, 'foo');
 <a id="ERR_INVALID_TUPLE"></a>
 ### ERR_INVALID_TUPLE
 
-An error with code `'ERR_INVALID_TUPLE'` is thrown when an element in the
-`iterable` provided to the [WHATWG][WHATWG URL API] [`URLSearchParams`
-constructor][`new URLSearchParams(iterable)`] does not represent a `[name,
-value]` tuple – that is, if an element is not iterable, or does not consist of
-exactly two elements.
+Used when an element in the `iterable` provided to the [WHATWG][WHATWG URL
+API] [`URLSearchParams`constructor][`new URLSearchParams(iterable)`] does not
```

Comment:
This issue is in master. I guess it should be addressed with a PR there that then gets backported to 8.x.